extensions/emmet/extension.toml

```diff
@@ -1,18 +1,19 @@
 id = "emmet"
 name = "Emmet"
 description = "Emmet support"
-version = "0.0.3"
+version = "0.0.4"
```

Comment:
Based on previous plugin PRs, we need to revert the version bump from here: that happens separately before the release.

Reply:
Is there anything further that I need to do at this time?